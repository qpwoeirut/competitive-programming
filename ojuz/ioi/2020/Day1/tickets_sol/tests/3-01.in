df804a0e-e4b1-4b8d-a0f5-83401a72b5a3
4 4 3
0 1 1 1
0 1 1 1
0 1 1 1
0 0 0 1
