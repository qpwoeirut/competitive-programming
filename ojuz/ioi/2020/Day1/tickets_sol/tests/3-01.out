131b432f-1f84-4f8f-9d55-41e868a6bc5d
OK
6
0 -1 1 2
1 -1 0 2
2 -1 0 1
0 1 2 -1
